postgres-driver/load_balancer.go

```diff
@@ -25,8 +25,12 @@ const (
 	VALUES (:lb_id, :app_id)`
 	updateLoadBalancer = `
 	UPDATE loadbalancers
-	SET name = COALESCE($1, name), user_id = COALESCE($2, user_id), updated_at = $3
```

Comment:
Are we sure `user_id` will never need to be updated?

Reply:
Yeah pretty sure since applications are always tied a single user ID.